index.html

```diff
@@ -61,5 +61,203 @@ <h3>I’am a Software Developer</h3>
       </div>
       <img src="asset/Group.png" alt="Group" class="i-image" />
     </section>
+    <main>
+      <section class="work-section">
+        <div class="caption flex">
+          <h2 class="caption">My Recent Works</h2>
+        </div>
+        <div class="cards grid">
+          <div class="card-details">
+            <img src="asset/card.png" alt="image" />
+            <div class="card-content on-hover">
+              <h3>Multi-Post Stories</h3>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <div class="btn-group flex">
+                <button class="btn">css</button>
+                <button class="btn">html</button>
+                <button class="btn">bootstrap</button>
+                <button class="btn">ruby</button>
+              </div>
+            </div>
+            <button class="special">See Project</button>
+          </div>

```

Comment:
- [ ] I've noticed that the `Recent works` section is missing a few touches. We understand that your design implementation does not need to be pixel perfect, but at least it should be a visual copy of the Figma designs. Please fix this before your next review.

Reply:
I only implemented the hover effect which was part of the requirements of this project as written on the activity page.

pls what are you expecting I’d like to understand